tests/python/cli/test_cli.py

```diff
@@ -362,7 +362,7 @@ def test_auto_annotate_with_threshold(self, fxt_new_task: Task):
         )
 
         annotations = fxt_new_task.get_annotations()
-        assert annotations.shapes[0].points[0] == 0.75
+        assert annotations.shapes[0].points[0] == pytest.approx(0.75)
```

Comment:
This isn't right. The value 0.75 comes from the CLI argument just above, and is supposed to be passed to `conf_threshold_function`, back out via the coordinate value, into the database, back out via the `/annotations` endpoint, and finally into the `annotations` variable. None of these steps is supposed to modify the value, so I do expect it to remain exactly equal to 0.75.

It's even a binary fraction, so it's exactly representable even in the single precision format; it should not be possible for it to be altered due to unexpected rounding.